run.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-./utils/js2key &
+./utils/js2key &&
```

Comment:
```suggestion
./utils/js2key &
```
在Linux里一个命令接一个`&`是表示让这条命令的**进程在后台运行**, 而`&&`是用于两个命令之间, 是**前一个命令执行成功才执行后一个命令的意思**, 是类似逻辑运算里的`&&`的用法, 与它类似的是两个命令间加`||`是表示**前一个命令执行失败才执行后一条命令**.

我这里用`&`是因为想要在`Soft-Robotics-ROV`这个程序运行的同时一直转发游戏手柄的操作, 所以得有一个程序在后台运行. 如果有两个终端倒不用这样.